include/MLInputGenerator.h

```diff
@@ -0,0 +1,41 @@
+// MLInputGenerator.h
+#ifndef mlinputgenerator_h
+#define mlinputgenerator_h 1
+
+#include "lcfiplus.h"
+#include "TTree.h"
+#include "TFile.h"
+#include "TVector3.h"
+
+#include <vector>
+#include <variant>
+
+namespace lcfiplus {
+namespace MLInputGenerator {
+
+  extern map<string, variant<
```

Comment:
Why not make this a class and mark the map as `static`? As far as I can tell that should have the same effect?